streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -182,7 +188,7 @@ public void writeAsText(String filePath, Serde<K> keySerde, Serde<V> valSerde) {
         try {
 
             PrintStream printStream = new PrintStream(new FileOutputStream(filePath));
-            topology.addProcessor(name, new KeyValuePrinter<>(printStream, keySerde, valSerde), this.name);
+            topology.addProcessor(name, new KeyValuePrinter<>(printStream, keySerde, valSerde, null), this.name);
```

Comment:
Should we extend (ie, new overloads) `writeAsText` with optional parameter `name` similar to `print()` ?